src/test/groovy/org/prebid/server/functional/tests/privacy/GppAmpSpec.groovy

```diff
@@ -0,0 +1,106 @@
+package org.prebid.server.functional.tests.privacy
+
+import org.prebid.server.functional.model.db.StoredRequest
+import org.prebid.server.functional.model.request.amp.AmpRequest
+import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.model.request.auction.Regs
+import org.prebid.server.functional.util.PBSUtils
+import org.prebid.server.functional.util.privacy.gpp.TcfEuV2Consent
+import org.prebid.server.functional.util.privacy.gpp.UspV1Consent
+
+import static org.prebid.server.functional.model.request.GppSectionId.TCF_EU_V2
+import static org.prebid.server.functional.model.request.GppSectionId.USP_V1
+import static org.prebid.server.functional.model.request.amp.ConsentType.GPP
+import static org.prebid.server.functional.model.response.auction.ErrorType.PREBID
+
+class GppAmpSpec extends PrivacyBaseSpec {
+
+    def "PBS should emit warning when consent_string is invalid"() {
+        given: "Default amp request with invalid consent_string"
+        def ampRequest = AmpRequest.defaultAmpRequest.tap {
+            consentString = PBSUtils.randomString
+            consentType = GPP
```

Comment:
This applies only to GPP consent type? Or to any consent type? Or it doesn't matter at all? This should most likely be a parametrized test.

Reply:
If we speak about scope of the test this warning applies only for `GPP` consent type.